recipes/bindash/build.sh

```
@@ -1,7 +1,25 @@
 #!/bin/bash
+
+export INCLUDES="-I${PREFIX}/include"
+export LIBPATH="-L${PREFIX}/lib"
+export LDFLAGS="${LDFLAGS} -L${PREFIX}/lib"
+
 mkdir release && pushd release
```

Shellcheck

[warning] Use 'pushd ... || exit' or 'pushd ... || return' in case pushd fails.

(SC2164)